cmd/system-probe/config/config_unsupported.go

```diff
@@ -3,7 +3,7 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-//go:build !unix && !windows
+//go:build !unix && !linux_bpf && !windows
```

Comment:
`unix` is inclusive of `linux`, so I don't think `!linux_bpf` is required.

Reply:
If we had `linux_bpf` but no `linux`, I think a lot of other things would break.

Reply:
removed